client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.jsx

```diff
@@ -1,17 +1,19 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import { useHistory, useLocation } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { Button } from 'reactstrap';
 import getFilePermissions from 'utils/filePermissions';
 import './DataFilesToolbar.scss';
 
-export const ToolbarButton = ({ text, iconName, onClick, disabled }) => {
+export const ToolbarButton = ({ text, iconName, onClick, disabled, id }) => {
   const iconClassName = `icon-action icon-${iconName}`;
   return (
     <Button
       disabled={disabled}
       onClick={onClick}
       className="data-files-toolbar-button"
+      id={id}
```

Comment:
@tacc-wbomar Can you comment as to whether this (id assignment to pick up [css class](https://github.com/TACC/Core-Portal/pull/549/files#diff-764b86f0476474e859d3d55857b419f3270cbe822ec3ea1af00b255a2ea08689R18)) is preferred over expanding the `classNames` for a component like this?

Reply:
Yup, I can. Nah, it's not.

Though `id` works, using IDs in CSS reduces reusability and is often overkill. Please try using `className` ([example](https://github.com/TACC/Core-Portal/blob/272ab4f/client/src/components/_common/LoadingSpinner/LoadingSpinner.jsx)).

<details><summary><strong>Suggestion</strong></summary>

Replace `#empty-button` with `.is-empty` and update accordingly.

<details>

To name CSS classes that show state, I've found `is-` and `has-` prefixes most legible. Examples:
- `.is-empty`
- `.has-children`
- `.is-collapsed`
- `:not(.is-collapsed)` (i.e. "not collapsed") (see [`:not()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:not))

</details>

</details>

<details><summary><strong>Reference</strong></summary>

[UI - CSS Style Guide - Specificity @ **IDs in CSS**](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Specificity#UICSSStyleGuideSpecificity-IDsinCSS)

</details>